.github/workflows/cargo-generate-test.yml

```diff
@@ -9,11 +9,18 @@ jobs:
       PROJECT_NAME: demo
       CARGO_GENERATE_VALUE_REGISTRY: ghcr.io
       CARGO_GENERATE_VALUE_REGISTRY_MODULE_PATH_PREFIX: "kubewarden/policies"
+      CARGO_GENERATE_VERSION: "v0.22.0"
```

Comment:
This is not the latest version. But it the last one that works. There is an issue if we try to generate using `v0.22.1` or later. I'll add an task in my notes to investigate that and open an issue if necessary.